web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
```

Comment:
I'm not sure how it's really supposed to go either. I've been keeping the following in mind: 
(from http://www.usgo.org/files/pdf/AGAMcMahonPairingInterface.pdf)
> "The program must also be able to record data for new AGA members that join at the tournament. Programs should automatically assign temporary AGA numbers in the range 90000-99999. Temporary numbers must be unique within each tournament. "